src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -1991,9 +2125,7 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
         // llvm::Value *p = CreateGEP(str, idx_vec);
         // TODO: Currently the string starts at the right location, but goes to the end of the original string.
         // We have to allocate a new string, copy it and add null termination.
-        llvm::Value *step = llvm::ConstantInt::get(context, llvm::APInt(32, 1));
-        llvm::Value *present = llvm::ConstantInt::get(context, llvm::APInt(1, 1));
-        llvm::Value *p = lfortran_str_slice(str, idx1, idx2, step, present, present);
+        llvm::Value *p = lfortran_str_copy(str, idx1, idx2);
```

Comment:
to be reverted (https://github.com/lcompilers/lpython/commit/2f45421c06c2bee6a9918ae71ca0b88530ebd6fd)